cnf-tests/testsuites/e2esuite/ptp/ptp.go

```diff
@@ -137,6 +138,79 @@ var _ = Describe("ptp", func() {
 		})
 	})
 
+	var _ = Describe("PTP socket sharing between pods", func() {
+		slaveLabel := ptpSlaveNodeLabel
+		BeforeEach(func() {
+			if discovery.Enabled() {
+				Skip("PTP socket test not supported in discovery mode")
+			}
+			_, slaveConfigs := getPTPConfigs(ptpLinuxDaemonNamespace)
+			if len(slaveConfigs) == 0 {
+				Skip("No nodes configured as ptp slaves found on the cluster")
+			}
+			slaveLabel = retrievePTPProfileLabels(slaveConfigs)
+			if slaveLabel == "" {
+				Skip("No nodes configured as ptp slaves found on the cluster: no node with PTP slave labels found")
+			}
+		})
+
+		AfterEach(func() {
+			err := namespaces.Clean(ptpLinuxDaemonNamespace, "testpod-", client.Client)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		var _ = Context("Run pmc in a new pod on the slave node", func() {
+			It("Create the pod on the slave node", func() {
```

Comment:
It() should contain what It(the component tested) should do or is expected to do.
to log steps you can use the By() function.

example:
It("should be able to communicate using a uds",func(){
    By("Creating a pod")
    /////////
    By("Executing pmc")
    /////////
})

Reply:
thanks updated :)